frontend/test/metabase/scenarios/filters/filter.cy.spec.js

```diff
@@ -200,29 +201,6 @@ describe("scenarios > question > filter", () => {
     cy.findByText(AGGREGATED_FILTER);
   });
 
-  it.skip("in a simple question should display popup for custom expression options (metabase#14341) (metabase#15244)", () => {
```

Comment:
Can this be removed? I know it was skipped, but not sure if this relates to the removed filter sidebar or not.

Reply:
unlike the sidebar, the modal isn't supposed to support custom expressions.